akka-stream/src/main/scala/akka/stream/scaladsl/Graph.scala

```diff
@@ -1292,15 +1292,15 @@ object GraphDSL extends GraphApply {
 
     @tailrec
     private[stream] def findOut[I, O](b: Builder[_], junction: UniformFanOutShape[I, O], n: Int): Outlet[O] = {
-      if (n == junction.outArray.length)
+      if (n == junction.outlets.length)
```

Comment:
what type is the `outlets` here? I think it does `_out :: Nil` (if that is the same outlet? `length` on List would not be good, unless it's only single element lists

Reply:
`outlets` is always a `Vector`, again due to an implementation detail of `FanOutShape`.

The `_out :: Nil` that you're referring to is on `FanInShape`, so that's not relevant here.